src/components/createTestCase/CreateTestCase.tsx

```diff
@@ -96,6 +95,18 @@ const CreateTestCase = () => {
 
   const handleSubmit = async (testCase: TestCase) => {
     setAlert(null);
+    const sanitizedDescription = DOMPurify.sanitize(testCase.description);
+    testCase = {
+      id: testCase.id,
+      title: testCase.title,
+      description: sanitizedDescription,
+      name: testCase.name,
+      series: testCase.series,
+      createdAt: testCase.createdAt,
+      createdBy: testCase.createdBy,
+      lastModifiedAt: testCase.lastModifiedAt,
+      lastModifiedBy: testCase.lastModifiedBy,
+    };
```

Comment:
Can be simplified with the spread operator. It will create a new object with the same values, then override any values specified afterwards.
```suggestion
    testCase = { ...testCase, description: sanitizedDescription};
```

Reply:
not sure if we even need to create a new object because we are reassigning to an existing variable.
just `testCase.description = DOMPurify.sanitize(testCase.description);` might be sufficient

Reply:
Thanks so much for the suggestions! They both work! I just changed the code to update only the field.